src/app/build.gradle

```diff
@@ -62,13 +45,19 @@ dependencies {
 	testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
 }
 
+test {
+	useJUnitPlatform()
+}
+
+jar{
```

Comment:
I have no idea why I needed to add this, but it was always being skipped, which meant I couldn't access app classes from userCLI...